test/uk/gov/hmrc/plasticpackagingtax/registration/views/TooManyAttemptsPasscodeViewSpec.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.views
+
+import base.unit.UnitViewSpec
+import org.scalatest.matchers.must.Matchers
+import uk.gov.hmrc.plasticpackagingtax.registration.views.html.too_many_attempts_passcode_page
+import uk.gov.hmrc.plasticpackagingtax.registration.views.tags.ViewTest
+
+@ViewTest
+class TooManyAttemptsPasscodeViewSpec extends UnitViewSpec with Matchers {
+
+  private val page = instanceOf[too_many_attempts_passcode_page]
+
+  "Too Many Attempts Passcode View" should {
+
+    "have proper messages for labels" in {
+      messages must haveTranslationFor("primaryContactDetails.sectionHeader")
+      messages must haveTranslationFor("primaryContactDetails.tooManyAttempts.passcode.title")
```

Comment:
I think there are some more messages which we could be checking for.

Reply:
Done